### DISCUSSION TITLE
Allow uploading zip files to Assistant.

### DISCUSSION THREAD

**Discussion by sathyarajv** [danny-avila/LibreChat#2168] [Q&A]
How to allow users to use zip files only with Assistant. Currently its not supported and getting an error message.

**Comment by danny-avila** [comment]
Haven't tested it but this is a quick fix. You're probably running into this issue:

![image](https://github.com/danny-avila/LibreChat/assets/110412045/ed811826-7c97-44cb-80f4-46baeff198f9)


Based off OpenAI Docs, I added `application/zip` not what you see in the image: https://platform.openai.com/docs/assistants/tools/supported-files

Will add it soon to the default list.

But as a workaround, you can make it work now and just use the fileConfig: https://docs.librechat.ai/install/configuration/custom_config.html#endpoint-file-config-object-structure
